pkg/agent/manager/cache/jwt_cache_test.go

```diff
@@ -221,6 +222,47 @@ func TestJWTSVIDCache(t *testing.T) {
 	}
 }
 
+func TestJWTSVIDCacheSize(t *testing.T) {
+	fakeMetrics := fakemetrics.New()
+	log, _ := test.NewNullLogger()
+	log.Level = logrus.DebugLevel
+	cache := NewJWTSVIDCache(log, fakeMetrics, 2)
+
+	now := time.Now()
+	jwtSvid1 := &client.JWTSVID{Token: "1", IssuedAt: now, ExpiresAt: now.Add(time.Second)}
+	jwtSvid2 := &client.JWTSVID{Token: "3", IssuedAt: now, ExpiresAt: now.Add(time.Second)}
+	jwtSvid3 := &client.JWTSVID{Token: "3", IssuedAt: now, ExpiresAt: now.Add(time.Second)}
+
+	spiffeID := spiffeid.RequireFromString("spiffe://example.org/blog")
+	cache.SetJWTSVID(spiffeID, []string{"audience-1"}, jwtSvid1)
+	cache.SetJWTSVID(spiffeID, []string{"audience-2"}, jwtSvid2)
+	cache.SetJWTSVID(spiffeID, []string{"audience-3"}, jwtSvid3)
+
+	_, ok := cache.GetJWTSVID(spiffeID, []string{"audience-1"})
```

Comment:
```suggestion
	// The first JWT-SVID should be evicted from the cache because we've added more JWT-SVIDs to the cache than the maximum configured size of the cache
	_, ok := cache.GetJWTSVID(spiffeID, []string{"audience-1"})
```
It would be helpful to leave a comment above this line about why this JWT-SVID is not expected to be in the cache